### PR TITLE
Sticky navigation with the burger icon AB test

### PR DESCRIPTION
**Pull request by Calanthe**
Another variation of the sticky navigation.

Desktop:
stickyburgerdesktop

Mobile:
stickyburgermobile

### CODE REVIEW

**Comment by kelvin-chappell** [comment]
:+1:

**Comment by piuccio** [comment]
why the padding around `all` only on mobile?

File: static/src/javascripts/projects/common/modules/experiments/tests/mt-sticky-burger.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+                config.$navigationScroll.css('display', 'block');
+                config.$navigationGreySection.css('border-top', '36px solid #00456e');
+            } else {
+                config.$navigationScroll.css('display', 'none');
+            }
+        }
+
+        function updatePosition(config) {
+            fastdom.write(function () {
+                if (window.scrollY < config.scrollThreshold) {
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/experiments/tests/mt-sticky-burger.js]: You are not testing it on IE but just FYI [scrollY](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY) is not working on IE.

File: static/src/javascripts/projects/common/modules/experiments/tests/mt-sticky-burger.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+                            updatePositionMobile(stickyConfig);
+
+                            mediator.on('window:scroll', _.throttle(function () {
+                                updatePositionMobile(stickyConfig);
+                            }, 10));
+                        } else {
+                            mediator.on('window:scroll', _.throttle(function () {
+                                //height of topAd needs to be recalculated because we don't know when we will get respond from DFP
+                                stickyConfig.stickyTopAdHeight = stickyConfig.$stickyTopAd.dim().height;
+                                updatePosition(stickyConfig);
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/experiments/tests/mt-sticky-burger.js]: I think this logic won't work when you come to the page through comments link.

**Comment by uplne** [comment]
Otherwise :+1:

**Pull request merged**